### ISSUE TITLE
getUniProtToIDMapping or getConvTable?

### ISSUE THREAD

**Issue by skiaphrene** [fmicompbio/einprot#25]
Hi, this isn't really an issue, but a call for clarification...
I don't think it's clear what the difference is between getUniProtToIDMapping() and getConvTable(). They clearly give different results, but I don't _understand_ the difference from the manual pages.

I am working with _C. elegans_, so if I call:
`einprot::getUniProtToIDMapping("roundworm")`
I get a data.frame mapping UniProt IDs to Gene Names / Symbols.

If I call:
`einprot::getConvTable(type = "WormBase") `
I get a data.frame mapping UniProt IDs to WormBase IDs. The latter has more mappings.

I need to use both, of course! :-)

Would it make sense to have a function that returned both mappings? Or a single function that could return either mapping? Or if the underlying code is so different, explain that in the help files, and add each function as a "See also" entry in the help file? Would it make sense to unify the argument specifications to each (e.g. "C. elegans" for both)? Just some ideas for clarification!

**Comment by csoneson** [comment]
Yes, fair point - the functions were originally written for different purposes, but I think it can indeed be simplified. I'll add it to the list.
